openapi.spec.yml

```diff
@@ -14,7 +14,7 @@ info:
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   version: 3.151.0
 servers:
-  - url: https://api.cf.eu12.hana.ondemand.com/
+  - url: https://api.cf.eu10.hana.ondemand.com/
```

Comment:
I'm about to switch to templated server urls, to support all environments.

https://swagger.io/docs/specification/api-host-and-base-path/